atk.env

```diff
@@ -2,5 +2,5 @@ DOCKERHUB_USERNAME=uwsbel
 
 ROS_DISTRO=humble
 
-DEFAULT_APT_DEPENDENCIES="bash vim git git-lfs python3-pip"
+DEFAULT_APT_DEPENDENCIES="bash vim git git-lfs python3-pip ros-humble-robot-localization"
```

Comment:
this should go in the atk.yml file and not here, so under your computers specific apt dependencies